lib/apipie/swagger_generator.rb

```diff
@@ -70,6 +70,7 @@ def init_swagger_vars(version, lang, clear_warnings=false)
               "x-copyright" => Apipie.configuration.copyright,
           },
           basePath: Apipie.api_base_url(version),
+          schemes: [ "http", "https" ],
```

Comment:
I wonder if this shouldn't be configurable? I guess not everyone is exposing their APIs via HTTP (if I understand correctly the reason for this change)